test/images/issue-74839/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM BASEIMAGE
```

Comment:
Seems like we will need boilerplate in dockerfile as well. Ref https://github.com/kubernetes/kubernetes/blob/master/test/images/netexec/Dockerfile